src/System.Private.ServiceModel/src/System/ServiceModel/Description/TypeLoader.cs

```diff
@@ -23,6 +23,7 @@ internal class TypeLoader
         private static Type[] s_messageContractMemberAttributes = {
             typeof(MessageHeaderAttribute),
             typeof(MessageBodyMemberAttribute),
+            typeof(MessagePropertyAttribute)
```

Comment:
>typeof(MessagePropertyAttribute) [](start = 12, length = 32)

Reviewers: the touches to product code were copied verbatim from the full framework.  These lines had been stripped out in our initial port because these attributes were not supported.